src/main/java/HW_1/Test3.java

```diff
@@ -0,0 +1,16 @@
+package HW_1;
+
+public class Test3 {
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 3;
+        a = a + b;
+        b = a - b;
+        a = a - b;
+        a = a + b - (b = a);
+        a = a ^ b;
+        b = b ^ a;
+        a = a ^ b;
```

Comment:
Do we really need all the line?
![Screenshot_5](https://github.com/AntBelyi/ALevel_automation/assets/66726924/56d24aeb-fe3c-445a-aaee-8cae854f4964)

Reply:
fixed